flask_rebar/utils/request_utils.py

```diff
@@ -92,7 +92,7 @@ def normalize_schema(schema):
     This allows for either an instance of a marshmallow.Schema or the class
     itself to be passed to functions.
     """
-    if not isinstance(schema, marshmallow.Schema):
+    if schema is not None and not isinstance(schema, marshmallow.Schema):
```

Comment:
```suggestion
    if issubclass(schema, marshmallow.Schema):
```

Reply:
Issue with this is that normalize_schema as it exists will accept:
1. an actual instantiated schema, in which case it just parrots back (those cases would cause an exception with `issubclass` absent some other checking to decide if the thing we have been handed is a class or an object)
2. anything invocable (in which case the current behavior is prob technically incorrect.. as it doesn't enforce that the invocable thing is a class and further a subclass of marshmallow.Schema, but since this has been in the wild for some time, conceivably there could be some evil genius [or lunatic, YMMV] out there passing some kind of generator function etc etc so I'm loathe to break it)

Reply:
Crap I forgot `issubclass` throws TypeErrors.
It seems like the docstring makes it fairly clear that the method is only intended to work with classes/class instances that derive from Marshmallow.Schema. 

I'd just throw the TypeError it it's not one of those:
```
if not isinstance(schema, marshmallow.Schema) and issubclass(schema, marshamallow.Schema):
```
Or `inspect.isclass` _should_ work if you want to avoid the TypeError.

But it does seems like there are a few equally valid ways you could take this. 🤔